mongofiles/mongofiles.go

```diff
@@ -66,14 +66,24 @@ type MongoFiles struct {
 	bucket *gridfs.Bucket
 }
 
+// SetupError is the error thrown by New to convey what error occurred and the appropriate exit code.
+type SetupError struct {
```

Comment:
This was copy/pasted from mongoexport; should it be moved to common? Or is it minor enough to just keep here?

Reply:
moved to common and filed TOOLS-2260